pytest.ini

```diff
@@ -1,6 +1,9 @@
 [pytest]
-addopts= -v --showlocals --durations 10
+addopts= -v --showlocals --durations 50
```

Comment:
Looks like most projects look for the slowest `10` tests here except `py-evm` since it has a lot of very different types of tests. Maybe we go with something like `20` here?

Honestly, take or leave... I just think it might add a lot of noise at the end of the test suite if we don't really care about the slowest `50` tests on most repos.

Reply:
changed it to 10 - you're right, mostly noise for most libs